### PR TITLE
Added polish translation

### PR DESCRIPTION
**Pull request by PiotrZPL**
As a native speaker of the Polish language and a contributor of this project, I added a translation for my language. It may not be perfect, but I don't see any problems with it so far.

### CODE REVIEW

File: src/translations/po/pl.po [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: plasma5-wallpapers-dynamic\n"
+"Report-Msgid-Bugs-To: https://github.com/zzag/plasma5-wallpapers-dynamic/issues\n"
+"POT-Creation-Date: 2020-06-20 12:58+0300\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Piotr Lange <zpl20052020@protonmail.ch>\n"
+"Language: \n"
```
Comment (zzag) [inline src/translations/po/pl.po:13]: nit: `Language: pl\n`

Comment (PiotrZPL) [reply src/translations/po/pl.po]: Just updated the file

**Comment by zzag** [comment]
It looks like this PR contains some unrelated commits (up to "[Merge branch 'zzag:main' into main](https://github.com/zzag/plasma5-wallpapers-dynamic/pull/85/commits/1745147249b36c9ba79dd5695a7bede880291e4e)")

Can you rebase this PR against the current main branch? e.g. (not tested)

```
git remote add upstream git@github.com:zzag/plasma5-wallpapers-dynamic.git
git fetch upstream
git rebase -i upstream/main
```

and remove all unrelated commits so there are only

* [Added polish translation](https://github.com/zzag/plasma5-wallpapers-dynamic/pull/85/commits/296b55020e68fd559a0c33b358bd0baf0f9333bc)
* [Update pl.po](https://github.com/zzag/plasma5-wallpapers-dynamic/pull/85/commits/dfc183964da175fcc08c6ecab8b5e30577b5b2d2)

while on this, you could also squash those two commits, however I can do that as well

**Comment by PiotrZPL** [comment]
Is this what I was supposed to do?

**Comment by zzag** [comment]
Thanks!

**Pull request merged**